waspc/data/Generator/templates/react-app/src/operations/resources.js

```diff
@@ -61,19 +62,14 @@ export async function invalidateAndRemoveQueries() {
  * @param {string[]} resources - Names of resources.
  */
 async function invalidateQueriesUsing(resources) {
-  const queryCacheKeysToInvalidate = getQueriesUsingResources(resources)
-    .filter(hasNoActionsInProgress)
-
   const queryClient = await queryClientInitialized
+
+  const queryCacheKeysToInvalidate = getQueriesUsingResources(resources)
   queryCacheKeysToInvalidate.forEach(
     queryCacheKey => queryClient.invalidateQueries(queryCacheKey)
```

Comment:
We call most of the variables representing query cache keys `queryCacheKey`. I think I'll rename all of them to `queryKey` (it's shorter and more inline with React Query's naming).

I didn't want to do it yet. Better to keep the diff clean until we decide on the approach.

Reply:
Sounds good!